### PR TITLE
Misc

### PR DESCRIPTION
**Pull request by lnussel**

### CODE REVIEW

**Comment by okurz** [comment]
LGTM

File: main.pm [hunk 1]
```diff
@@ -719,6 +719,9 @@ sub load_x11tests() {
         && !system_is_livesystem)
     {
         loadtest "x11/gnucash.pm";
+        loadtest "x11/xchat.pm";
+        loadtest "x11/hexchat.pm";
+        loadtest "x11/vlc.pm";
```
Comment (nilxam) [inline main.pm]: do you check xchat and vlc are installed by default x11/gnome/kde pattern? they're on the online repo but I didn't see those are trying to install inside of test, ie. ensure_installed. so if they aren't installed in default pattern the tests won't work.

Comment (nilxam) [inline main.pm]: just check the media I owned here, not the last snapshot release, vlc does not in the media, hexchat and xchat does.

**Comment by nilxam** [comment]
LGTM

**Comment by DimStar77** [comment]
Let's just drop the xchat test please: the package has just been discussed and flagged for removal. no need to test it

**Comment by lnussel** [comment]
It's still in Leap though so potential candidate for maintenance testing. Once dropped from TW we can add a check_version

**Pull request merged**